htdocs/PI/write/PIWriteRequest.php

```diff
@@ -556,17 +556,27 @@ private function checkIfGOCDBIsReadOnlyAndRequestisNotGET(){
 
 
     /**
-     * Gets authentication information and sets the relevant class property
+     * Sets the relevant class property
      */
-    private function getAndSetAuthInfo() {
+    private function getAndSetAuthInfo($authArray) {
```

Comment:
If this no longer gets authentication information, the function should be renamed.

Reply:
Agreed, fixed in latest commit